libcxx/test/libcxx/containers/associative/multimap/non_cv_object.verify.cpp

```diff
@@ -0,0 +1,47 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// Checks that we can only form multimaps of object types.
+
+#include <map>
+
+std::multimap<int const, int> K1;
+std::multimap<int, int const> M1;
+// TODO(cjdb): turn this into a compile-time error
```

Comment:
Instead of adding TODOs here, could you file a Github issue listing the cases you'd like to tackle as a follow-up?

Reply:
How about both? I like doing `// TODO(#<issue-number>)` since it means one can identify exactly where the issue is referring to, and how many problems there are to solve before closing.

Reply:
I've opened #106635.